### DISCUSSION TITLE
How to show 8,000,000 at Maximum of LinearAxis.

### DISCUSSION THREAD

**Discussion by sakai-IST** [oxyplot/oxyplot#1941] [Q&A] [answered]
I'm Sorry if this is the wrong place to start the discussion.

![キャプチャ](https://user-images.githubusercontent.com/98733014/192452428-2fe4d25b-8fcc-4eb7-a60b-be5f39206c67.PNG)
Is it possible to show 8,000,000 as the maximum value on the Y axis?
In my Example(this picture), the maximum value show '1e + 08'.

**Comment by VisualMelon** [comment, marked as answer]
Change the `StringFormat` property for your axis, or set a custom `LabelFormatter`. The default string format is `"g6"`, which switches to scientific notation at 7 figures:

1
10
100
1000
10000
100000
1e+06
1e+07
1e+08
1e+09
1e+10

See https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings for information on other numeric format strings

**Reply by sakai-IST** [reply]
Thanks for your reply and sorry my late reply.
I just tried your suggestion and it worked! thank you!
